pipeline/backend/backend.go

```diff
@@ -3,20 +3,22 @@ package backend
 import (
 	"fmt"
 
+	"github.com/urfave/cli/v2"
```

Comment:
would be nice if we could pass backend options somehow else into pipeline/** packages

Reply:
had an idea, will push